src/Engine/babylon.engine.ts

```diff
@@ -5252,6 +5252,8 @@
             this._bindTextureDirectly(bindTarget, texture, true);
 
             this._uploadDataToTextureDirectly(texture, texture.width, texture.height, imageData, faceIndex, lod);
+
+            this._bindTextureDirectly(bindTarget, null, true);

```

Comment:
This will have the same issue as this morning with the instance demo
Code assume that texture is still bind after loading it

Reply:
Nope as I changed the upload method accordingly to uploadDirectly (no bind/unbind) like in DDS:
https://github.com/BabylonJS/Babylon.js/pull/4690/files#diff-bad50cfda6c5c1f8fe4bba23866c0667L199

This was a typo of the previous PR :-(

I added the according test for TGA so that I am not breaking it again... I hope ;-)

Reply:
Can you just make sure and run the instance démo as well?

Reply:
Yup Triple Checked :-)

![image](https://user-images.githubusercontent.com/16655165/42351543-6c266452-806a-11e8-8ee9-c57f710534e0.png)